### DISCUSSION TITLE
align column widths across multiple rows in bootstrap form

### DISCUSSION THREAD

**Discussion by theumi** [twbs/bootstrap#35091] [Q&A]
In a form, I would like to align column widths across multiple rows. In the individual rows I use input-groups and a number input.

`
      <form class="row g-1">  
        <div class="input-group mb-0">
          <span class="input-group-text" id="1A">A</span>
          <span class="input-group-text" id="1A">Ich habe gerne neue Situationen und entwickle so viele Lösungsmöglichkeiten wie möglich.</span>
          <input type="number" class="form-control" min="0" max="10" step="1" aria-label="1A" aria-describedby="1A" required>
        </div>
        [...]  
`

As drawn in red in the following image, I would like to set the column alignment.

![grafik](https://user-images.githubusercontent.com/7475619/135620570-2f0ec9b3-fb23-460b-99ea-810143e8b1cb.png)

With individal CSS width statements I've found a solution. Is there an overarching simpler solution? I am grateful for a tip.

**Comment by adamplesnik** [comment]
Try this: `.w-75` does the trick.
```
<div class="input-group my-2">
  <span class="input-group-text w-75" id="1A">
    <span class="pr-3">
      1A
    </span>
    <span class="text-truncate d-inline-block">Ich habe gerne neue Situationen und entwickle so viele Lösungsmöglichkeiten wie möglich.</span>
  </span>
  <input type="number" class="form-control" min="0" max="10" step="1" aria-label="1A" aria-describedby="1A" required>
</div>
```

More at this fiddle: https://jsfiddle.net/qsecj9ya/
